src/Aspire.Dashboard/DashboardWebApplication.cs

```diff
@@ -12,19 +12,25 @@
 using Aspire.Dashboard.Otlp.Grpc;
 using Aspire.Dashboard.Otlp.Storage;
 using Microsoft.AspNetCore.Authentication.Certificate;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.HttpsPolicy;
 using Microsoft.AspNetCore.Server.Kestrel.Core;
 using Microsoft.AspNetCore.Server.Kestrel.Https;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.FluentUI.AspNetCore.Components;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
 
 namespace Aspire.Dashboard;
 
 public class DashboardWebApplication : IAsyncDisposable
 {
-    internal const string DashboardInsecureAllowAnonymousVariableName = "DOTNET_DASHBOARD_INSECURE_ALLOW_ANONYMOUS";
-    internal const string DashboardOtlpAuthModeVariableName = "DOTNET_DASHBOARD_OTLP_AUTH_MODE";
-    internal const string DashboardOtlpApiKeyVariableName = "DOTNET_DASHBOARD_OTLP_API_KEY";
+    internal const string WebAppAuthModeKey = "DashboardWebApp:AuthMode";
```

Comment:
I want to put all the dashboard config in a `Dashboard` container.
```suggestion
    internal const string FrontendAuthModeKey = "Dashboard:Frontend:AuthMode";
```

Reply:
Is it idiomatic that an app makes its own namespace in its own config file? I don't mind either way. Perhaps you can do that in #3119 after this merges.

Reply:
It doesn't need to be in its own config file. You can mix settings in config files. This happens in appsettings.json.

Yes, it's normal to have configuration for a thing sit under its own section. For example, all of Kestrel's settings are under `Kestrel`, all of logging's settings are under `Logging`, etc.